gpm_api/tests/test_utils/test_utils_checks.py

```diff
@@ -0,0 +1,741 @@
+import numpy as np
+from typing import Union
+import xarray as xr
+
+import pytest
+from pytest_mock import MockerFixture
+
+from gpm_api.utils import checks
+from utils import convert_hours_array_to_datetime_array
+
+
+# Utility functions ###########################################################
+
+
+def create_dataset_with_coordinate(coord_name: str, coord_values: np.ndarray) -> xr.Dataset:
+    """Create a dataset with a single coordinate"""
+
+    ds = xr.Dataset()
+    ds[coord_name] = coord_values
+    return ds
+
+
+def create_orbit_time_array(time_template: Union[list, np.ndarray]) -> np.ndarray:
+    """Create a time array with ORBIT_TIME_TOLERANCE as unit"""
+
+    start_time = np.datetime64("2020-12-31T00:00:00")
+    time = np.array([start_time + checks.ORBIT_TIME_TOLERANCE * t for t in time_template])
+    return time
+
+
+@pytest.fixture
+def set_is_orbit_to_true(
+    mocker: MockerFixture,
+) -> None:
+    mocker.patch("gpm_api.checks.is_orbit", return_value=True)
+    mocker.patch("gpm_api.checks.is_grid", return_value=False)
+    mocker.patch("gpm_api.utils.checks.is_orbit", return_value=True)
```

Comment:
Is it necessary to also mock the import in the ` gpm_api.utils.checks` to patch correctly? 
I wonder if should not put all these checks utility into a single module (or create a directory checks ... and create various modules for the various checks tools)

Reply:
Yes, it did not work without mocking in `gpm_api.utils.checks.*`, even thought `is_orbit` and `is_grid` were imported from `gpm_api_checks`. For now, I moved the fixture and utils in `conftest.py` and `gpm_api/tests/test_utils/utils.py`